packages/Webkul/Shop/src/Routes/api.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use Webkul\Shop\Http\Controllers\CartController;
+use Webkul\Shop\Http\Controllers\ProductController;
+use Webkul\Shop\Http\Controllers\Customer\Account\CompareController;
+use Webkul\Shop\Http\Controllers\Customer\WishlistController;
+use Webkul\Shop\Http\Controllers\CategoryController;
+
+Route::group(['middleware' => ['locale', 'theme', 'currency'], 'prefix' => 'api'], function () {
+
+    Route::controller(ProductController::class)->group(function () {
+        Route::get('category-products/{categoryId}', 'getByCategory')
+            ->name('shop.category_products.get');
+    });
+
+    Route::controller(CategoryController::class)->group(function () {
+        Route::get('categories/filter-attributes/{id?}', 'getFilterAttributes')
```

Comment:
GET `categories/{id}/attributes` CategoryController@getAttributes

The remaining filterable flag will be handled from the query param.